src/entrypoints/array minimal.ts

```diff
@@ -0,0 +1,2 @@
+import './external-scripts-loader'
```

Comment:
is that a space in the file name rather than a dot? just noticed this after approving but I'm guessing it should be a dot?

Reply:
i don't think so... or I can't see it

Reply:
sorry I meant in `src/entrypoints/array minimal.ts`, not in `./external-scripts-loader` 🙃